doc/topics/development/deprecations.rst

```diff
@@ -20,14 +20,14 @@ Salt feature releases are based on the Periodic Table. Any new features going
 into the ``master`` branch will be named after the next element in the Periodic
 Table. For example, Magnesium was the feature release name associated with the
 ``v3002`` tag. At that point in time, any new features going into the
-``master`` branch, after ``v3002`` was tagged, were part of the Aluminium feature
```

Comment:
There shouldn't be any changes in this file since it's giving examples of how to deprecate something with the feature release name. Can you change back the references to aluminium in this file.